server/api/api.py

```diff
@@ -0,0 +1,306 @@
+from aiohttp import web
+import asyncio
+
+from api.nodes.nodes import list_nodes, install_node, delete_node
+from api.models.models import list_models, add_model, delete_model
+from api.settings.settings import set_twilio_account_info
+from pipeline import Pipeline
+
+from comfy.nodes.package import _comfy_nodes, import_all_nodes_in_workspace
+
+from api.nodes.nodes import force_import_all_nodes_in_workspace
+#use a different node import
+import_all_nodes_in_workspace = force_import_all_nodes_in_workspace
+
+def add_routes(app):
+    app.router.add_get("/env/list_nodes", nodes)
+    app.router.add_post("/env/install_nodes", install_nodes)
+    app.router.add_post("/env/delete_nodes", delete_nodes)
+
+    app.router.add_get("/env/list_models", models)
+    app.router.add_post("/env/add_models", add_models)
+    app.router.add_post("/env/delete_models", delete_models)
+
+    app.router.add_post("/env/reload", reload)
+    app.router.add_post("/env/set_account_info", set_account_info)
```

Comment:
minor nit: would suggest the root path to be `/settings` or `/config` instead of `/env`.